django/otto/views.py

```diff
@@ -707,13 +707,13 @@ def cost_dashboard(request):
     # Filter by dates
     if date_group == "last_90_days":
         start_date = timezone.now().date() - timedelta(days=90)
```

Comment:
So that the total time is actually 90, the start date should also be advanced 1 day.
(and same for the others)

Reply:
Updated to reflect the accurate duration. Oversight on my part. Thank you.